### ISSUE TITLE
Upload multiple pictures at a time

### ISSUE THREAD

**Issue by pullopen** [Plume-org/Plume#837]
**Is your feature request related to a problem? Please describe.**
Plume now only support uploading one picture a time, and it would be very convenient if it support uploading multiple pictures at a time. We can arrange the name or so later. 

**Describe the solution you'd like**
Supporting mutiple pictures at a time would be very convenient. The picture name can be set defaultly as file name. And if it allows multiple selection in the gallery and then show the markdown code at the same time, it would be better. 

**Additional context**
Plume: 0.6.1-dev

**Comment by KitaitiMakoto** [comment]
Yep, those are nice features. I want them, too. Will start to development after some current implementations completed.
